src/components/Footer/Footer.js

```diff
@@ -305,8 +217,8 @@ const styles = StyleSheet.create({
   btnComment: (isShowDM, isSelf) => ({
     height: '100%',
     justifyContent: 'center',
-    paddingLeft: isShowDM && !isSelf ? 20 : 13.5,
-    paddingRight: isShowDM && !isSelf ? 6 : 10
+    paddingLeft: 13.5,
+    paddingRight: 10
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The padding for the `btnComment` style has been changed. This will affect the visual appearance of the comment button. Ensure that these changes align with your design requirements.
```diff
-    paddingLeft: isShowDM && !isSelf ? 20 : 13.5,
-    paddingRight: isShowDM && !isSelf ? 6 : 10
+    paddingLeft: 13.5,
+    paddingRight: 10
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->